test/utils/create_resources.go

```diff
@@ -51,7 +51,16 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc) error {
 }
 
 func IsRetryableAPIError(err error) bool {
-	return apierrs.IsTimeout(err) || apierrs.IsServerTimeout(err) || apierrs.IsTooManyRequests(err) || utilnet.IsProbableEOF(err)
+	// These errors may indicate a transient error that should be retried.
```

Comment:
s/should be retried/in tests we can retry/

Reply:
Done.